lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -292,7 +292,8 @@ sub locate_local_assets {
         my $file = locate_asset($assetkeys->{$key}, $vars->{$key}, mustexist => 1);
         unless ($file) {
             my $error = "Cannot find $key asset $assetkeys->{$key}/$vars->{$key}!";
-            return {error => $error};
+            log_error("$key handling $error", channels => 'autoinst');
+            return {error => $error} unless $key eq 'UEFI_PFLASH_VARS';
```

Comment:
At some point i guess we will have some exception list :)

Reply:
Actually I want to create a ticket for this, we have this asset handling in 3-4 different places, including caching... and well, we know where this path leads to... also, to remove these workarounds...